.github/workflows/relayer.yml

```diff
@@ -24,6 +24,16 @@ jobs:
 
       - name: Unit Test
         run: nix develop -c make test-unit-go
+      
+      - name: Upload Golangci relayer results
+        if: always()
```

Comment:
Should not be needed if always:
```suggestion
```

Reply:
Isn't always necessary to ensure we still run on failure? Default is success only IIRC